### ISSUE TITLE
Set MiniDrawer under toolbar

### ISSUE THREAD

**Issue by akhbulatov** [mikepenz/MaterialDrawer#2271]
For tablet mode, I create a `MiniDrawer` in the root activity and add 2 toolbars in the left and right fragment. The problem is that I need `MiniDrawer` to be under the left toolbar, not on the same level as it is now. How can this be achieved?

<img width="866" alt="2018-04-11 12 28 25" src="https://user-images.githubusercontent.com/6171938/38608395-e56a3872-3d83-11e8-8cab-4790e6003b60.png">

**Comment by mikepenz** [comment]
@akhbulatov you control where you add the view of the `MiniDrawer` just change where in your view hierarchy you add it :D

**Comment by akhbulatov** [comment]
In my case, there are 2 `FrameLayout` containers in the root activity for the left and right fragment, in which the root `LeftFragment` and `RightFragment` are added. Inside them, other fragments with toolbars will add/replace as child fragments. When I create `MiniDrawer` in RootActivity, I pass an instance of `RootActivity`. Then where should I add `MiniDrawer` that it appears under toolbar of left child fragment? Isn't it in `RootActivity`?

**Comment by mikepenz** [comment]
@akhbulatov that's completely in your control and depends on the architecture of your application. 

In the `PrsistentDrawerActivity` the whole content is on the right side of the `Crossfader`
https://github.com/mikepenz/MaterialDrawer/blob/develop/app/src/main/res/layout/activity_persistent_drawer.xml#L8

**Comment by akhbulatov** [comment]
I was able to place `MiniDrawer` under toolbar of a left child fragment by setting view's id that is under toolbar in layout. But now when I slide the `MiniDrawer`, the left fragment hides behind the right fragment instead of sliding it to right, since I set it as content for `Crossfader`. How to make the left fragment is not hiding behind the right one, and slides with it to right?

<img width="865" alt="2018-04-14 21 15 35" src="https://user-images.githubusercontent.com/6171938/38771345-4ed89e0e-402a-11e8-9a27-53dd87bb7e97.png">
